chat-plugins/othermetas.js

```diff
@@ -128,10 +128,24 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		if (!toId(target) || !target.includes('@')) return this.parse('/help mixandmega');
 		let sep = target.split('@');
-		let stone = Object.assign({}, Dex.getItem(sep[1]));
-		let template = Object.assign({}, Dex.getTemplate(sep[0]));
-		if (!stone.exists || (stone.exists && !stone.megaEvolves && !stone.onPrimal)) return this.errorReply(`Error: Mega Stone not found`);
-		if (!template.exists) return this.errorReply(`Error: Pokemon not found`);
+		let stone = toId(sep[1]);
+		if (toId(sep[1]) === 'dragonascent') {
+			stone = {
+				id: "dragonascent",
+				name: "Dragon Ascent",
+				megaStone: "Rayquaza-Mega",
+				megaEvolves: "Smeargle",
```

Comment:
In retrospect, it makes no sense to make these changes when there are tentative /mnm changes on main already. When your PR gets merged I'll restart this